packages/gatsby-theme-api-docs/src/components/resource/method/method.js

```diff
@@ -49,6 +66,26 @@ const Method = ({
   if (method.uriParameters) {
     allUriParameters = allUriParameters.concat(method.uriParameters);
   }
+
+  let contentType;
+  if (method.body) {
+    const findOutContentTypes = Object.keys(method.body).reduce(
+      (list, value) => {
+        return method.body[value] !== null ? [...list, value] : [...list];
+      },
+      []
+    );
+    findOutContentTypes.forEach((type) => {
+      if (!contentType) {
+        contentType = convertContentType(type);
+      } else {
+        contentType = `${contentType} ${convertContentType(type)}`;
+      }
+    });
+  }
```

Comment:
my feeling is that this whole contentType discovery could be somehow simplified but I'm not getting the full logic behind it.
If you have time I'd recomend you extract this to a separate utils function, document it with a couple of lines of descriptions and maybe add few unit test just to make sure that any future refactor will not break.

Reply:
The root cause lies in the design of the data structure in the GraphQL surface (IMHO).